demo/demo-spring-boot-transport/demo-spring-boot-pojo-server/src/main/resources/microservice.yaml

```diff
@@ -8,3 +8,11 @@ cse:
       address: http://127.0.0.1:30100
   rest:
     address: 0.0.0.0:8080
+  handler:
+    chain:
+      Provider:
+        default: myhandler
+servicecomb:
```

Comment:
if you want to test auto urlPattern, should this be deleted?

Reply:
```yaml
servicecomb:
  rest:
    servlet:
      urlPattern: /*
```
I guess you mean the configuration above should be deleted, right?